controllers/policies/policyserver_controller.go

```diff
@@ -162,6 +163,7 @@ func (r *PolicyServerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				policy.Status.PolicyStatus = policiesv1alpha2.ClusterAdmissionPolicyStatusUnscheduled
 				err := r.Reconciler.UpdateAdmissionPolicyStatus(context.Background(), policy)
 				if err != nil {
+					metrics.RecordPolicyCount(policy)
```

Comment:
I'm not sure about this line; I feel it should be out of the `if err!= nil` block. I also feel that a fresh new unscheduled policy is going to be counted twice: here in this line and then later on.
If the intent is to update the policy with the RecordPolicyCount(), so the fields are also updated, could it be possible to call `metrics.RecordPolicyCount()` only once, just inside of `UpdateAdmissionPolicyStatus()`?

Reply:
Oh... you are right. Moving the function call to `UpdateAdmissionPolicyStatys` is much better.